v2/services/authorize/credentialsmanager/managerV1.go

```diff
@@ -0,0 +1,62 @@
+package credentialsmanager
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/secretsmanager"
+	"github.com/pkg/errors"
+
+	"github.com/SKF/go-utility/v2/log"
+)
+
+type SMAPIV1 interface {
+	GetSecretValue(input *secretsmanager.GetSecretValueInput) (*secretsmanager.GetSecretValueOutput, error)
+}
+
+type CredentialsManagerV1 struct {
```

Comment:
Just a naming thing, this should probably be called e.g. `credentialsFetcherV1` now?